Monika After Story/game/updates.rpy

```diff
@@ -372,6 +372,11 @@ label v0_3_1(version=version): # 0.3.1
     return
 
 # non generic updates go here
+#0.12.0
+label v0_12_0(version="v0_12_0"):
+    python:
+        persistent.ever_won["nou"] = False
```

Comment:
do we really need an update script? can we just do these kinds of sets in `ch30_reset` if "nou" isn't in the dict?

Reply:
We could do it in `ch30_reset`, but why? We need to do it only once.

Reply:
actually rather than ch30 reset we should move this to the init code for nou. If it's always checked for on startup then its more robust since we don't have to rely on code that only runs once. Especially with the chess update script bug we had.

We should also follow this style for future games as well - games are more like extra modules so they should handle updating/adding entries into the relevant dictionaries they need to be in without relying on run-once code.